### DISCUSSION TITLE
DatePicker always returns Null

### DISCUSSION THREAD

**Discussion by AllTheUserNamesAreGone** [filamentphp/filament#12256] [Help]
### Package

Panel builder

### Package Version

3

### How can we help you?

Hi,

I'm trying to use the DatePicker as shown here:

```
  DatePicker::make('date')
                            ->native(false)
                            ->closeOnDateSelection()
                            ->displayFormat('d M Y')
                            ->default(now())
                            ->prefixIcon('heroicon-o-calendar-days')
                            ->format('Y-m-d')
                            ->required()
                            ->label('Purchase Date')
                            ->columnSpan(1),
```

This always returns Null unless I comment out or remove ``` ->native(false) ```. 

Any ideas why this might be?  Thanks for reading this far.

**Comment by leandrocfe** [comment]
Where is always returning null?

**Reply by AllTheUserNamesAreGone** [reply]
I don't know what you mean. 

Do you mean how do I know it's always turning null? If so, because it always fails validation saying it is a required field and when I do a dd() is shows ```"date" => null ```

If you mean where's the form - it's in a Resource file.
